sdk/eventhub/event-hubs/src/partitionPump.ts

```diff
@@ -187,7 +187,7 @@ export function createProcessingSpan(
   });
 
   span.setAttributes({
-    component: "eventhubs",
+    "az.namespace": "Microsoft.EventHub",
```

Comment:
Where do these names come from? Should it be 'Microsoft.EventHub(s)'?

Reply:
these are Azure resource namespaces https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers

There is motivation here https://github.com/Azure/azure-sdk-for-js/issues/6702